parsers/c/tokens.ml

```diff
@@ -112,6 +112,7 @@ type token =
   | PIPE_EQ
   | COMMA
   | LBRACK_LBRACK
+  | EQ_EQ_GT
```

Comment:
Please delete this and add a `CN_IMPLIES` underneath `CN_WILD`.

Reply:
Should I also include `CN_IMPLIES -> "CN_IMPLIES"` under `string_of_token`?